src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedParameterSymbol.cs

```diff
@@ -227,35 +230,39 @@ public override ImmutableArray<CustomModifier> RefCustomModifiers
 
         private sealed class SynthesizedComplexParameterSymbol : SynthesizedParameterSymbolBase
         {
-            private readonly ImmutableArray<CustomModifier> _refCustomModifiers;
-            private readonly ImmutableArray<CSharpAttributeData> _attributes;
+            private readonly ParameterSymbol _baseParameter;
```

Comment:
nit: consider enabling nullability analysis in this file, or asserting that `baseParameter` is not null in constructor.